src/Module/Update/Network.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+namespace Friendica\Module\Update;
+
+use Friendica\Core\System;
+use Friendica\DI;
+use Friendica\Model\Item;
+use Friendica\Module\Conversation\Network as NetworkModule;
+
+class Network extends NetworkModule
+{
+	public static function rawContent(array $parameters = [])
+	{
+		if (!isset($_GET['p']) || !isset($_GET['item'])) {
+			exit();
+		}
+
+		self::parseRequest($parameters, $_GET);
+
+		$profile_uid = intval($_GET['p']);
+
+		$o = '';
+
+		if (!DI::pConfig()->get($profile_uid, 'system', 'no_auto_update') || ($_GET['force'] == 1)) {
+			if (!empty($_GET['item'])) {
```

Comment:
At several places in the code we now do have the "update" calls (in network, community, contacts, ...) with a given `item`. The system always has to do the same: Return the thread that carries an item with this id.

I think we should create a single function that is called by all these update functions. And important: The id does not need to be the parent. it can be any item in this thread if I remember it correctly. This depends on the defifferent themes.

Reply:
Like you can see it's only half a dozen lines, so I don't think it would benefit much from being in a single function. Besides, I feel like this PR is big enough as it is, if you want to move forward with this function, please do so in a follow-up.